Formula/parquet-tools.rb

```diff
@@ -0,0 +1,28 @@
+class ParquetTools < Formula
+  desc "ApacheParquet command-line tools and utilities"
+  homepage "https://parquet.apache.org/"
+  env :std
```

Comment:
Don't need this, and we wouldn't accept a new formula that needs `env :std`. The problem you're trying to solve I suspect can be resolved by adding `ENV.java_cache` at the top of `def install`.